sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetScan.scala

```diff
@@ -1148,9 +1151,9 @@ case class GpuParquetPartitionReaderFactory(
     }
     new ParquetPartitionReader(conf, file, singleFileInfo.filePath, singleFileInfo.blocks,
       singleFileInfo.schema, isCaseSensitive, readDataSchema,
-      debugDumpPrefix, maxReadBatchSizeRows,
-      maxReadBatchSizeBytes, metrics, singleFileInfo.isCorrectedInt96RebaseMode,
-      singleFileInfo.isCorrectedRebaseMode, singleFileInfo.hasInt96Timestamps, readUseFieldId)
+      debugDumpPrefix, maxReadBatchSizeRows, maxReadBatchSizeBytes, targetSizeBytes, metrics,
+      singleFileInfo.isCorrectedInt96RebaseMode, singleFileInfo.isCorrectedRebaseMode,
+      singleFileInfo.hasInt96Timestamps, readUseFieldId)
```

Comment:
Seems like the non-multi reader also needs to know whether to use the chunked reader?